uniffi_macros/src/export/attributes.rs

```diff
@@ -226,6 +303,14 @@ impl ExportedImplFnAttributes {
                     }
                     this.constructor = true;
                 }
+                "method" => {
+                    if this.constructor {
```

Comment:
do we really need this - maybe this can just be left unchecked and this becomes a noop?